src/main/java/com/hubspot/jinjava/el/ext/RegularDivOperator.java

```diff
@@ -0,0 +1,46 @@
+package com.hubspot.jinjava.el.ext;
+
+import de.odysseus.el.misc.TypeConverter;
+import de.odysseus.el.tree.impl.Parser.ExtensionHandler;
+import de.odysseus.el.tree.impl.Parser.ExtensionPoint;
+import de.odysseus.el.tree.impl.Scanner;
+import de.odysseus.el.tree.impl.ast.AstBinary;
+import de.odysseus.el.tree.impl.ast.AstBinary.SimpleOperator;
+import de.odysseus.el.tree.impl.ast.AstNode;
+
+public class RegularDivOperator extends SimpleOperator {
+  public static final Scanner.ExtensionToken TOKEN = new Scanner.ExtensionToken("/");
+  public static final RegularDivOperator OP = new RegularDivOperator();
+
+  @Override
+  protected Object apply(TypeConverter converter, Object a, Object b) {
+    if (a == null || b == null) {
+      throw new IllegalArgumentException("Division argument may not be null");
+    }
+
+    // Debugging support only -- Delete me
```

Comment:
TODO: don't forget to delete!

Reply:
Deleted

Reply:
I did have a question earlier on, when I was assuming that` RegularDivOperator ` should return a long when dividing long arguments and that it should return a double when dividing double or mixed arguments. When I tried coding `RegularDivOperator` that way I got a regression failure on the `ExpressionResolverTest.itResolvesMathVals()`, which was expecting a result of "5.0" when dividing 15L by 3L. Switching to having `RegularDivOperator` always return a double made the question about how to simply and reliably format the result vanish. It also fixed the regression and yielded a clean build.